packages/cli-kit/src/private/node/ui/alert.tsx

```diff
@@ -0,0 +1,68 @@
+import {Banner, BannerType} from '../components/Banner.js'
+import {Link} from '../components/Link.js'
+import {List, ListItem} from '../components/List.js'
+import {renderOnce} from '../ui.js'
+import {consoleLog, consoleWarn, Logger, LogLevel} from '../../../output.js'
+import React from 'react'
+import {Box, Text} from 'ink'
+
+export interface AlertProps {
+  type: Exclude<BannerType, 'error'>
+  headline?: string
+  body: string
+  nextSteps?: ListItem[]
+  reference?: ListItem[]
+  link?: {
+    label: string
+    url: string
+  }
+  orderedNextSteps?: boolean
+}
+
+const typeToLogLevel: {[key in AlertProps['type']]: LogLevel} = {
+  info: 'info',
+  warning: 'warn',
+  success: 'info',
+}
+
+const typeToLogger: {[key in AlertProps['type']]: Logger} = {
+  info: consoleLog,
+  warning: consoleWarn,
+  success: consoleLog,
+}
+
+export function alert({type, headline, body, nextSteps, reference, link, orderedNextSteps = false}: AlertProps) {
+  renderOnce(
```

Comment:
What about being consistent with the other modules and have modules representing the components:
```
private/
  node/
    ui/
      components/
         Alert.tsx
         Error.tsx
```